source/beamline/AMControl.cpp

```diff
@@ -54,3 +54,11 @@ bool AMControl::hasChildControl(AMControl *control) const
 
 	return result;
 }
+
+void AMControl::removeChildControl(AMControl *control)
```

Comment:
This implementation is alike with the ones in the subclass `AMPseudoMotorControl`. except that the subclass called `updateState()` after the child is removed.

I'm wondering whether we should change the signature of this function to return a bool value. Then in the subclass (`AMPseudoMotorControl`), we can call 
```
if (AMControll::removeChildControl(control)) { 
    updateStates()
}
````